### DISCUSSION TITLE
Completion within in-line latex block?

### DISCUSSION THREAD

**Discussion by joshcho** [company-mode/company-mode#1298] [Q&A] [answered]
How can I complete within $$ block for org mode? Company completes just fine in () block, but I would love it if completion worked within $$ blocks.

**Comment by dgutov** [comment, marked as answer]
Org-specific completion logic should live in Org. I don't think there are separate backend for it.

You can try forcing completion with the built-in UI using `C-M-i`. If you doesn't do what you wanted, it's a good cause to file a feature request for Org. No need to even mention Company in that scenario.
